Assets/MXR.SDK/package.json

```diff
@@ -17,6 +17,7 @@
 		}
 	],
 	"dependencies": {
+		"com.cysharp.unitask": "2.5.10",
```

Comment:
Have we tested the pipeline with this change?

Reply:
Good catch! I'll check. Local builds are working so it's likely CD should too. The custom-launcher repo still uses 2.2.5 so we might want to update it to this version anyway